instrumentation/grizzly-2.0/javaagent/src/test/groovy/GrizzlyTest.groovy

```diff
@@ -58,6 +58,11 @@ class GrizzlyTest extends HttpServerTest<HttpServer> implements AgentTestTrait {
     server.stop()
   }
 
+  @Override
+  boolean hasResponseCustomizer(ServerEndpoint endpoint) {
```

Comment:
Seems like there's a naming conflict here... perhaps this should be renamed to `hasResponseMutator` at some point?

Reply:
We should rename the `HttpServerResponseCustomizerHolder` as well